### PR TITLE
Move mongo config away from cors package

### PR DESCRIPTION
**Pull request by rodrigok**

### CODE REVIEW

File: packages/rocketchat-mongo-config/server/index.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import { Mongo } from 'meteor/mongo';
+
+import tls from 'tls';
+// FIX For TLS error see more here https://github.com/RocketChat/Rocket.Chat/issues/9316
+// TODO: Remove after NodeJS fix it, more information https://github.com/nodejs/node/issues/16196 https://github.com/nodejs/node/pull/16853
+tls.DEFAULT_ECDH_CURVE = 'auto';
```
Comment (sampaiodiego) [inline packages/rocketchat-mongo-config/server/index.js]: do we still need this? seems node have fixed it already.

this is not related to mongo btw, so I don't it should be in this package =) but my vote is to remove.

Comment (rodrigok) [reply packages/rocketchat-mongo-config/server/index.js]: Done
